python/ctsm/toolchain/gen_mksurfdata_jobscript_multi.py

```diff
@@ -5,11 +5,18 @@
 """
 import os
 import sys
-import argparse
+import logging
 
-from ctsm.path_utils import path_to_ctsm_root
```

Comment:
@samsrabin this is the other script to go over.

Reply:
This is straight forward because it's mostly removing things in order to use the subroutines from single.

Reply:
As part of this I did remove the capability of setting the queue (which isn't needed for Derecho). single does set the queue to a proper default for each machine listed. One issue that was there was that the queue is different depending on the machine, and the list of queues also varies. So this would need more infrastructure to support.

So removing it for now seemed like the best approach.

Reply:
In issues #2446 and #2439